keylime/tpm/tpm_main.py

```diff
@@ -870,6 +869,8 @@ def verify_ek(self, ekcert, ekpem):
             ekcertpem = ek509.get_pubkey().get_rsa().as_pem(cipher=None).decode('utf-8')
 
             # Make sure given ekcert is for their ek
+            print('ekpem: ', str(ekpem))
```

Comment:
Was this left in for debugging? Should it be sent to the logger instead?

Reply:
ack, this is debug residue.